src/compose/ComposeMenu.js

```diff
@@ -212,48 +232,28 @@ export default function ComposeMenu(props: Props): Node {
   const styles = useMemo(
     () =>
       createStyleSheet({
-        composeMenu: {
+        container: {
+          marginTop: 2, // So it doesn't run into the text input
```

Comment:
Let's put more spacing here. Say, 4px. (That's what you see in my screenshot in the previous comment.)

Reply:
Hmm, with these buttons not being on their own panel, do you think they want to be vertically centered between the bottom of the message input and the bottom of the compose box? Here's with 4px:

<img width="418" alt="image" src="https://user-images.githubusercontent.com/22248748/211882202-7a079fc2-cb1f-46c5-995a-ac371c5a9e31.png">

How about 8px:

<img width="418" alt="image" src="https://user-images.githubusercontent.com/22248748/211882312-f73da7af-bee9-450b-a68b-f7037ad17d15.png">

Then if that seems too loose and we want a denser layout, one idea is to change that and all the other 8px padding/margins in the compose box to be 4px instead (which, it's hard to tell, might go in the direction of Vlad's screenshot?):

<img width="400" alt="image" src="https://user-images.githubusercontent.com/22248748/211883221-9df1df27-201b-462a-99a1-e7286c9b1622.png">

Another is to take the 8px distance below and above the bottons(' touchable areas) and just make it zero:

<img width="400" alt="image" src="https://user-images.githubusercontent.com/22248748/211885474-669d71ed-a8d2-4b55-b764-0353b0ceac0a.png">